### DISCUSSION TITLE
Suggesting a small tweak to the Leveling Tracker - Exp Part

### DISCUSSION THREAD

**Discussion by FadeDeklin** [Lailloken/Exile-UI#347] [Ideas & Feedback]
Hello,
I am writing to suggest a minor tweak about effective exp gained in the current zone,
I wonder if you can make it not only show the exp gain in percentage but also show the levelling safe zone instead.

Example:
Player LVL: 30
Area LVL: 35
Display: -5(Max 4) 95.86%    ""(Text in red color becoz outside safe range)""

Player LVL: 33
Area LVL: 27
Display: +6(Max 5) 96.18% ""(Text in red color becoz outside safe range)""

The idea is to show precisely how many lvl you are under/over, the live max safe zone number, I don't need to calculate by myself,
so that you can be more manageable in fighting or running over the zone instead of just knowing I am still gaining 100% or not.

**Comment by Lailloken** [comment]
Thanks for your suggestion. I did some quick testing to see what it looks like:

| char lvl 2, area lvl 6 | char lvl 12, area lvl 6 |
|---|---|
| ![image](https://github.com/Lailloken/Lailloken-UI/assets/61888437/27fd589f-b16c-48f5-9aa4-cfc752512f77) | ![image](https://github.com/Lailloken/Lailloken-UI/assets/61888437/df4accfb-cf21-4c0b-b09b-7521ff98bede) |

For space reasons, I chose to express the lvl discrepancy a bit differently. The value in parentheses simply reflects how far off your level is to receiving no exp-penalty (i.e. "under/overleveled by X levels"). What do you think, and do the values make sense?

**Reply by FadeDeklin** [reply]
This makes sense and is excellent, but sadly, we cannot state the actual safe zone range due to less space. (Cry)
Love you.

**Reply by Lailloken** [reply]
How do you use the safe-zone (and adjust to it) while leveling? Just asking because I don't take many things except e-exp % into consideration while leveling (because I usually use Oni-Goroshi, which is like cheating).

**Comment by Lailloken** [comment]
Changed my mind again for v1.52.4, and I added additional padding if the text inside the guide-panel is rather short, so now the exp-panel should always have enough space.
While within the safe-zone, the panel will display ` min area-level for 100% | current area-level | max area-level for 100% `, so for a level 13 character in a level 16 area it shows: ` 10 | 16 | 16 `.
